src/sentry/static/sentry/app/views/performance/transactionDetails/quickTrace.tsx

```diff
@@ -299,13 +303,26 @@ function EventNodeSelector({
           const target = generateSingleEventTarget(event, organization, location);
           return (
             <DropdownItem key={event.event_id} to={target} first={i === 0}>
-              <Truncate
-                value={event.transaction}
-                maxLength={35}
-                leftTrim
-                trimRegex={/\.|\//g}
-                expandable={false}
-              />
+              <DropdownItemSubContainer>
+                <Projects orgId={organization.slug} slugs={[event.project_slug]}>
+                  {({projects}) => {
+                    const project = projects.find(p => p.slug === event.project_slug);
+                    return (
+                      <ProjectBadge
+                        hideName
+                        project={project ? project : {slug: event.project_slug}}
+                        avatarSize={16}
+                      />
```

Comment:
Can we add the project badge to the single child tooltips too?

Reply:
Do we have the pattern where we add a project icon to tooltips? I have never seen this.

Reply:
Ah no you're right 🤔 I think it'd be nice if we could see project for single events too, but I won't holdup this PR for that